app.js

```diff
@@ -1,57 +1,252 @@
+    pEl.appendChild(ulEl);
+
   }
-]
+};
+
+var bellevueSqStore = {
+  storeName: 'Bellevue Square Store',
+  minCust: 20,
+  maxCust: 48,
+  avgCookie: 3.3,
+  totals: [],
+  grandTotal: 0,
+  hours: [
+    '10am: ',
+    '11am: ',
+    '12pm: ',
+    '1pm: ',
+    '2pm: ',
+    '3pm: ',
+    '4pm: ',
+    '5pm: '
+  ],
 
-stores.randCust =  function(min, max) {
-  return Math.floor(Math.random() * (max - min + 1) + min);
+  getRandom: function() {
+    return Math.random() * (this.maxCust - this.minCust) + this.minCust;
+  },
+
+  hourTotal: function() {
+    for(var j = 0; j < this.hours.length; j++) {
+      var ran = Math.floor(this.getRandom() * this.avgCookie);
+      this.totals.push(ran);
+      this.grandTotal += ran;
+    }
+  },
+
+  render: function() {
+    this.hourTotal();
+    var pEl = document.getElementById('belVue');
+    var ulEl = document.createElement('ul');
+
+    for (var i = 0; i < this.hours.length; i++) {
+      var liEl = document.createElement('li');
+      pEl.textContent = this.storeName;
+      liEl.textContent = this.hours[i] + this.totals[i];
+      ulEl.appendChild(liEl);
+    }
+    var liMent = document.createElement('li');
+    liMent.textContent = 'TOTALS ' + this.grandTotal;
+    ulEl.appendChild(liMent);
+    pEl.appendChild(ulEl);
+  }
 };
 
-stores.render = function() {
-  var kV = Object.keys(this);
-  var newUl = document.createElement('ul');
-  for (var i = 0; i < kV.length; i++) {
-    var newLi = document.createElement('li');
-    newUl.appendChild(newLi);
-    var newText = document.createTextNode(kV[i]);
+var alkiStore = {
+  storeName: 'Alki Store',
+  minCust: 3,
+  maxCust: 24,
+  avgCookie: 2.6,
+  totals: [],
+  grandTotal: 0,
+  hours: [
+    '10am: ',
+    '11am: ',
+    '12pm: ',
+    '1pm: ',
+    '2pm: ',
+    '3pm: ',
+    '4pm: ',
+    '5pm: '
+  ],
+
+  getRandom: function() {
+    return Math.random() * (this.maxCust - this.minCust) + this.minCust;
+  },
+
+  hourTotal: function() {
+    for(var j = 0; j < this.hours.length; j++) {
+      var ran = Math.floor(this.getRandom() * this.avgCookie);
+      this.totals.push(ran);
+      this.grandTotal += ran;
+    }
+  },
+
+  render: function() {
+    this.hourTotal();
+    var pEl = document.getElementById('alki');
+    var ulEl = document.createElement('ul');
+
+    for (var i = 0; i < this.hours.length; i++) {
+      var liEl = document.createElement('li');
+      pEl.textContent = this.storeName;
+      liEl.textContent = this.hours[i] + this.totals[i];
+      ulEl.appendChild(liEl);
+    }
+    var liMent = document.createElement('li');
+    liMent.textContent = 'TOTALS ' + this.grandTotal;
+    ulEl.appendChild(liMent);
+    pEl.appendChild(ulEl);
   }
-}
+};
 
-for (store in stores) {
-  stores.render();
+var stores = [
+  pikePlaceStore,
+  alkiStore,
+  seaTacStore,
+  bellevueSqStore,
+  southCenterStore
+];
+for(var k = 0; k < stores.length; k++) {
+  stores[k].render(stores[k].grandTotal);
 }
-
-stores.render();
```

Comment:
If you haven't already, you can make the hours array a global object so you don't have to list them for each store.